src/bin/commands/aws-cdk-version.test.ts

```diff
@@ -0,0 +1,7 @@
+import { AwsCdk } from "./aws-cdk-version";
+
+describe("AwsCdk", () => {
+  it("should list the aws-cdk libraries being used", () => {
```

Comment:
I don't think this is the best description of this test. What I'm trying to test is can we read from `package.json` - if `AwsCdk.ALL_VERSIONS` has at least 1 item, then we can.